script/Arcueid/Day 8/08_00_ARC08_2.txt

```diff
@@ -77,17 +76,17 @@ I take a sip of the black tea and lock eyes with Akiha.
 [sha:1e3449b17982929cd210648f7ed25e507786cb1d]{
 -- Page 58, Offset 19824.
 -- あぶないあぶない。むせるむせる。
-That was bad. I almost choked there.
+Not good―――choking, I'm choking.
 }
 [sha:67c5a81ffdd420c382da05de40f44a609e821462]{
 -- Page 58, Offset 19825.
 -- 秋葉のヤツ、初手から直球とは待ったなしにも程がある。
-That Akiha went too far. She didn't even wait before throwing a straight at the first pitch.
+Damn it Akiha, you're not supposed to go throwing fastballs right out of the gate.
```

Comment:
maybe drop 'Akiha'?

Reply:
Eh, I kinda feel like we need the subject reference since there otherwise isn't one on this page?